packages/plugin-expo-eas-sourcemaps/package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
-  "peerDependencies": {
+  "dependencies": {
```

Comment:
Why is this needed?

Reply:
sorry, left over from testing. should be `peerDependency`